### PR TITLE
update v2 delete doc

### PR DESCRIPTION
**Pull request by michaellzc**

### CODE REVIEW

**Comment by michaellzc** [comment]
Current dependencies on/for this PR:
* main
  * **PR #5724** <a href="https://app.graphite.dev/github/pr/sourcegraph/handbook/5724" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/sourcegraph/handbook/5724?utm_source=stack-comment).

File: content/departments/cloud/technical-docs/v2.0/delete_process.md [hunk 1]
```diff
@@ -70,39 +70,27 @@ cd environments/$ENVIRONMENT/deployments/$INSTANCE_ID/
...
+mi2 instance edit --query '.spec.debug.enableDeletionProtection = false' --slug $SLUG -e $ENVIRONMENT
+mi2 generate cdktf --slug $SLUG -e $ENVIRONMENT
 cd environments/$ENVIRONMENT/deployments/$INSTANCE_ID/terraform/stacks/sql
 terraform init
-export SQL_RESOURCE=$(terraform state list | grep sql_self)
-echo "$(jq '.resource.'$SQL_RESOURCE' += {"delete_protection":false}' cdk.tf.json)" > cdk.tf.json
 terraform apply -auto-approve
 ```
 
-### Removes GKE backups and restores
```
Comment (unknwon) [inline content/departments/cloud/technical-docs/v2.0/delete_process.md:93]: We don't need to remove these anymore?

**Review by unknwon — APPROVED** [review]
Assuming the last deletion part is intended

File: content/departments/cloud/technical-docs/v2.0/delete_process.md (hunk 1 above)
Comment (michaellzc) [reply content/departments/cloud/technical-docs/v2.0/delete_process.md:93]: > We don't need to remove these anymore?

these steps were extremely error-prone, so we replace them with `mi2 instance destroy --slug $SLUG -e $ENVIRONMENT -auto-approve`

https://github.com/sourcegraph/handbook/pull/5724/files#diff-7b1d9ed5a07c946916c213841da752199a404bf3cb978d9d6d795154b151f942R76-R77

**Pull request merged**